src/profile/ChangePassword.js

```diff
@@ -11,91 +11,91 @@ import LoaderButton from "../components/LoaderButton";
  * @TODO refactor into React Hooks
  */
 
-export default class ChangePassword extends Component {
-  constructor(props) {
-    super(props);
+const ChangePassword = (props) => {
 
-    this.state = {
-      password: "",
-      oldPassword: "",
-      isChanging: false,
-      confirmPassword: "",
-    };
-  }
+  const [state, setstate] = useState({
+    password: "",
+    oldPassword: "",
+    isChanging: false,
+    confirmPassword: "",
+  });
 
-  validateForm() {
+  const validateForm = () => {
     return (
-      this.state.oldPassword.length > 0 &&
-      this.state.password.length > 0 &&
-      this.state.password === this.state.confirmPassword
+      state.oldPassword.length > 0 &&
+      state.password.length > 0 &&
+      state.password === state.confirmPassword
     );
-  }
+  };
 
-  handleChange = (event) => {
-    this.setState({
-      [event.target.id]: event.target.value,
+  const handleChange = (event) => {
+    setstate({
+      ...state,
+      [event.target.id]: event.target.value
     });
   };
 
-  handleChangeClick = async (event) => {
+  const handleChangeClick = async (event) => {
     event.preventDefault();
-
-    this.setState({ isChanging: true });
+    setstate({
+      ...state,
+      isChanging: true
+    });
 
     try {
       const currentUser = await Auth.currentAuthenticatedUser();
-      await Auth.changePassword(
-        currentUser,
-        this.state.oldPassword,
-        this.state.password
-      );
-      this.props.history.push("/settings");
-    } catch (e) {
+      await Auth.changePassword(currentUser,state.oldPassword,state.password);
```

Comment:
I am noticing here that there is no space between the currentUser, state.oldPassword, state.password. It's not incorrect, the code will run, but we want to enforce the linting that is used throughout the project. Would you mind installing the Prettier es-lint extension in VS Code (assuming you are using VS Code) and then make sure that you either a) run it before you make a commit on a particular file or b) have it run automatically, whenever you save a file.

Reply:
I have installed the extension but it asks for adding dev dependencies to work along, I can add them or you can tell me if there a way around this solution. I use VS code too. As of now, I have made the changes manually but will surely look for ways to automate the formatter for future submissions 🙂

Reply:
Was it this extension? https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode I'm not exactly sure by what you mean when you say 'dev dependencies', it's been a long time since I've installed it myself. For this PR, it's all good - thank you for making the changes manually. Try to install the dev dependencies in the future, and see what happens - likely, it's just some internal VS code config that is being download.